rmgpy/molecule/symmetryTest.py

```diff
@@ -76,6 +76,31 @@ def testAtomSymmetryNumberEthane(self):
                 symmetryNumber *= calculateAtomSymmetryNumber(molecule, atom)
         self.assertEqual(symmetryNumber, 9)
     
+    def testAtomSymmetryNumberEthanewithDeuteriumTritium(self):
```

Comment:
more complicated molecules (based on RMG's imagination) might need to be tested. Can we justify the entropy changes for the molecules from eg1 in RMG-tests (https://travis-ci.org/ReactionMechanismGenerator/RMG-tests/builds/268177870?utm_source=github_status&utm_medium=notification)? They seem to differ by `R*ln2`. But I'm not fully clear about why the chiral center will half the symmetry number on top of previous symmetry calculation. Do you foresee any potential issues with kinetic estimation for a reaction with one side having chiral center?